docs/cloud/dbaas/postgres-accessing.md

```diff
@@ -0,0 +1,155 @@
+# Access your PostgreSQL database
+
+See [the page on firewalls](firewalls.md) for instructions on opening access to your database.
+
+## Graphical user interface
+
+A popular tool for working with PostgreSQL databases is pgAdmin, which can be found [here](https://www.pgadmin.org/). It can be installed as a desktop application, or on a server to be accessed via its web GUI. Note that it can not be installed on the Pukki database instance, and the DBaaS team does not provide support for it, as we are more comfortable using the CLI tools.
+
+## Command-line
+
+1. Install the `postgresql` command line tool. Note that some Linux distributions might provide ancient versions of it by default. Please consult [this page](https://www.postgresql.org/download/) for detailed installation instructions.
+2. Find the public IP of your database instance from its Overview tab in the web GUI, or with `openstack database instance list`.
+3. Use the following commands to access your PostgreSQL instance from the command line:
+   ```
+   psql --user ${USERNAME} --host ${PUBLIC_IP} ${DATABASE_NAME}
+   ```
+
+   The syntax normally used in configuration files is:
```

Comment:
which files? it should be clearer to mention the normal config file, or at least give some examples of names and locations.

Reply:
I think I clarified the text.